proto/v1/sql_service.proto

```diff
@@ -384,6 +384,7 @@ message CheckRequest {
     DDL = 1;
     DDL_GHOST = 2;
     DML = 3;
+    SQL_EDITOR = 4;
```

Comment:
DQL maybe?

Reply:
Sadly, this enum is scoped to our usage of SQL Editor instead of broader DQL per-se.